client/js/views/AddOffering.js

```diff
@@ -0,0 +1,41 @@
+module.exports = Object.assign({}, require('./__proto__'), {
```

Comment:
Update the indentation in this file to 4 spaces.

I've come to the conclusion using `eslint` and `prettier` are helpful tools to organize the source code automatically.  Let me know if you'd like to spend time helping me implement this in our application.

Reply:
Hmm on my computer it does look like the tab indent is 4 spaces.  Are there specific lines that you were referring to?

Also, I would definitely like to set up eslint and prettier when you have time!

Reply:
If you look at the diff [here](https://github.com/FutureDaysSoftware/FarmWebsite/pull/10/files/2a5e9d47be67e6c553d873e6b18611d3deef8efb#diff-9a4c89903e207eeafdfe0f4706ad2c66) you can see that the formatting is not 4 spaces.

Reply:
Go ahead and get started with a minimal example: https://eslint.org/docs/user-guide/getting-started

https://prettier.io/docs/en/configuration.html